music21/romanText/tsvConverter.py

```diff
@@ -155,18 +290,20 @@ def tabToM21(self):
 
         >>> tabCd = romanText.tsvConverter.TabChord()
         >>> tabCd.numeral = 'vii'
+        >>> tabCd.representationType = 'm21'
         >>> tabCd.global_key = 'F'
         >>> tabCd.local_key = 'V'
-        >>> tabCd.representationType = 'm21'
         >>> m21Ch = tabCd.tabToM21()
 
         Now we can check it's a music21 RomanNumeral():
 
-        >>> m21Ch.figure
+        # >>> m21Ch.figure
         'vii'
         '''
 
-        if self.numeral:
+        if self.numeral == '@none':
+            thisEntry = harmony.NoChord()
+        else:
```

Comment:
what if self.numeral is None?  -- this seems to have taken away a possibility w/o comment